spec/parser/ruby/ruby_parser_spec.rb

```diff
@@ -283,6 +283,58 @@ def hello; end
       end
     end
 
+    it "\n     Parses several formats of percent arrays into source strings\n" \
+         "     in constant declarations.  Loops thru all combinations of\n" \
+         "     () {} [] <> || delimiters, %i %I %w %W array types, and five\n" \
+         "     types of element/delimiter whitespace." do
+      delimiter_h = {
```

Comment:
Testing each delimiter is unnecessary. There is no codepath for delimiters in any of the parser code. Testing with a single delimiter should be sufficient-- if `%w()` worked and `%w{}` failed, that would not be due to a bug in YARD's codebase.